lib/cog/models/ecto_json.ex

```diff
@@ -73,13 +104,24 @@ defmodule Cog.Models.EctoJson do
   defp do_render(%{__struct__: Ecto.DateTime}=ts, _render_policy) do
     ts
   end
+  defp do_render(%{__struct__: Cog.Models.Permission.Namespace}=struct, render_policy) do
+    fields = Cog.Models.Permission.Namespace.__json_fields__(render_policy)
+    Enum.reduce(fields, "",
+      fn(field, acc) ->
+        if field == :name do
+          Map.get(struct, field)
+        else
+          acc
+        end
+      end)
+  end
```

Comment:
I think you can get by without the `reduce` here since you're overwriting `acc` anything you find a `field` called `:name`. Would this work the same?

```
fields = Cog.Models.Permission.Namespace.__json_fields__(render_policy)

if :name in fields do
  Map.get(struct, :name)
else
  ""
end
```

Still seems like a weird implementation but I don't really know the specifics here.